addons/web/static/src/public/colibri.js

```diff
@@ -283,6 +306,7 @@ export class Colibri {
                     throw new Error(`Invalid directive: '${directive}'${suffix}`);
                 }
             }
+            return toReplay;
```

Comment:
What happens for the next selector or the loop ? Shouldn't the array be defined outside the loop ?